libsource/src/main/java/com/virtusize/libsource/ui/VirtusizeInPageStandard.kt

```diff
@@ -332,7 +332,7 @@ class VirtusizeInPageStandard(context: Context, attrs: AttributeSet) : Virtusize
      * Sets up the InPage Standard footer margins
      */
     private fun getCloudinaryImageUrl(cloudinaryPublicId: String): String {
-        return "https://res.cloudinary.com/virtusize/image/upload/t_product-large-retina-v1/$cloudinaryPublicId.jpg"
+        return "https://res.cloudinary.com/virtusize/image/upload/q_auto,f_auto/$cloudinaryPublicId.jpg"
```

Comment:
Same as iOS:

you should provide the exact size that you need to reduce the bandwidth, that can be done by adding [`w_XX,h_XX`](https://cloudinary.com/documentation/image_transformations#img_trans_overview) to the query

You can use [`dpr_auto`](https://cloudinary.com/documentation/image_optimization#responsive_image_sizing) to ensure that images are still optimized for the device

Reply:
@zr0z I changed it to `https://res.cloudinary.com/virtusize/image/upload/w_${36.dpInPx},h_${36.dpInPx}/q_auto,f_auto,dpr_auto/$cloudinaryPublicId.jpg` in the commit a98bf52